frontend/homework/3/prog1.js

```diff
@@ -0,0 +1,23 @@
+function tipCalculator(bill){
```

Comment:
The function names must be proper, as you are using this function for both the tip and the total amount calculation